### DISCUSSION TITLE
TLS certificate is being provisioned. This may take up to 15 minutes to complete (5th version of this question)

### DISCUSSION THREAD

**Discussion by ginagiannetti** [community/community#22364] [Other Feature Feedback, Questions, & Ideas]
<p>TLS certificate is being provisioned. This may take up to 15 minutes to complete</p>
<p>Sorry this question has been asked before it’s just that replies seem to be ignored.</p>
<p>I am having the above error in settings and I cannot seem to get a certificate for my web page <a href="https://gina.dev" rel="noopener nofollow ugc">https://gina.dev</a> - looking at the other queries it seems that I need someone to refresh it on the back end please?</p>
<p>Many thanks</p>

**Comment by jfullstackdev** [comment]
<p>check this out,</p>
<aside class="quote quote-modified" data-post="1" data-topic="216832">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/github-pages-isnt-hosting-my-organizations-page/216832">Github Pages isn't hosting my organization's page</a> <a class="badge-wrapper  bullet" href="/c/github-help/github-pages/31"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #c8e1ff;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Discussion and questions about GitHub Pages—websites for you and your projects hosted right on your GitHub repository.">GitHub Pages</span></a>
  </div>
  <blockquote>
    Hello, I have uploaded the build of a Gatsby.js website to Github Pages that you can see in this repo <a href="https://github.com/Exsite-Studio/Exsite-Studio" rel="noopener nofollow ugc">https://github.com/Exsite-Studio/Exsite-Studio</a> 
And configured the pages settings in the repo as you can see in the screenshot attached 
 <a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/a/ba0a6a347f468e2d5be8cbd6a4bfaa4642cb30d5.png" data-download-href="/uploads/short-url/qxN7DKUGTAFooD40um00Kzdmdi5.png?dl=1" title="Screen Shot 2021-12-10 at 3.11.21 PM" rel="noopener nofollow ugc">[Screen Shot 2021-12-10 at 3.11.21 PM]</a> 
In the past, the website was working properly. But, right now it’s not even accessible at <a href="https://exsite-studio.github.io" rel="noopener nofollow ugc">exsite-studio.github.io</a> 
Any help would be appreciated! 
Thanks!
  </blockquote>
</aside>

**Comment by jfullstackdev** [comment]
<p>maybe it’s your DNS config also</p>

**Comment by ginagiannetti** [comment]
<p>Hi the DNS is configured as advised, please see attached<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/1/314d1ccb17b7bb2f3b67dca7e5bdd8f2ccdce6f6.png" data-download-href="/uploads/short-url/728BK5RkUtHqvwCqManhc18uC5U.png?dl=1" title="2021-12-13 19_49_30-Advanced DNS" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/43302007/181070558-a00c1b2d-c440-4b92-ba26-2a306f74836f.png" alt="2021-12-13 19_49_30-Advanced DNS" data-base62-sha1="728BK5RkUtHqvwCqManhc18uC5U" width="690" height="235" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/1/314d1ccb17b7bb2f3b67dca7e5bdd8f2ccdce6f6_2_690x235.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/1/314d1ccb17b7bb2f3b67dca7e5bdd8f2ccdce6f6_2_1035x352.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/1/314d1ccb17b7bb2f3b67dca7e5bdd8f2ccdce6f6_2_1380x470.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/1/314d1ccb17b7bb2f3b67dca7e5bdd8f2ccdce6f6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">2021-12-13 19_49_30-Advanced DNS</span><span class="informations">1723×589 35.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ginagiannetti** [comment]
<p><a class="mention" href="/yoannchaudet">@yoannchaudet</a> are you able to help with this please?</p>

**Comment by yoannchaudet** [comment]
<p>Hello <a class="mention" href="/ginadev">@ginadev</a>,</p>
<p>Please remove <code>162.255.119.191</code> from your A record. This is not a GitHub Pages IP address and this is preventing us to request a TLS certificate for your domain.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/d/2dbf11f3562e34a5d329caddb144c39e3141f8b5.png" data-download-href="/uploads/short-url/6wGQP8v1h0ynfzger7Lp4R3IYFD.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/14911070/181070560-2efb6cf0-885f-4827-bb61-2723e16e795a.png" alt="image" data-base62-sha1="6wGQP8v1h0ynfzger7Lp4R3IYFD" width="690" height="439" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/d/2dbf11f3562e34a5d329caddb144c39e3141f8b5_2_690x439.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/d/2dbf11f3562e34a5d329caddb144c39e3141f8b5_2_1035x658.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/d/2dbf11f3562e34a5d329caddb144c39e3141f8b5.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/d/2dbf11f3562e34a5d329caddb144c39e3141f8b5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1278×814 95.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Once it is fixed, the provisioning should go smoothly.</p>
<p>Best,</p>

**Comment by ginagiannetti** [comment]
<p>Thanks very much, I don’t have that in my list though… I sent a screenshot above</p>

**Comment by ginagiannetti** [comment]
<p>ah I spoke to them and found it, there was a URL redirect in my DNS servers that was causing it, many thanks</p>

**Reply by tfaile** [reply]
I had the same issue with namecheap's URL redirect, but mine was pointing to a different IP: 192.64.119.150.  I only saw this IP address in this dig command:

```
dig [domain] +noall +answer -t A
```
The tricky thing about this was that the namecheap admin interface did not show the 192.64.119.150 IP address.  I suspected that it must be the culprit from this thread (thanks!), so I deleted the URL Redirect Record along with the other good A and CNAME records in namecheap's admin interface, added the good records back, saved changes in namecheap, and in a couple of minutes the Github Pages interface finally provisioned the certificate and let me click the Enforce HTTPS.

**Comment by yoannchaudet** [comment]
<p>It looks like an IP address belonging to Namecheap. It is the second time this happened to someone. I am sorry it confuses people. As you say, I assume they automatically put this in the DNS zone to handle URL redirections.</p>
<p>I just checked and your certificate is in order now :tada:</p>

**Comment by MrDanielHarka** [comment]
I am now having the same issue. I have tried the followings:
- removing the custom domain and adding it again
- changing the custom domain to a different one then changing it back to the desired one
- deleting the repo and creating it again
- leaving **[it](https://github.com/MrDanielHarka/bookmarks.harka.com)** alone for 16 hours

It always issued my TLS certificate in 5-10 minutes, but now nothing seems to work.
I have done it quite a few times before and have no idea what might be the issue now.

![bookmarks-github-tls-issue](https://user-images.githubusercontent.com/79216552/196023573-d68eef9a-7e75-4a78-9d2a-b6ad6f6636c1.jpg)

**Reply by Riley-** [reply]
~I am having a similar issue, stuck at 'TLS certificate is being issued'. If I remove and re-enter the custom domain I can occasionally get this to work through waiting a while afterward, but otherwise it gets stuck. In my case I am generating the CNAME through a Github action workflow, and up until recently this was working as expected.~

Edit: Looks like the certificates are being issued as normal again, at least in my case

**Comment by WalterUpgrade** [comment]
<img width="796" alt="Captura de pantalla 2025-01-30 a las 13 06 30" src="https://github.com/user-attachments/assets/e6d0f2bf-ef02-4f5c-bbf4-07037ea40166" />
I still have the same problem with GoDaddy custom domain.
